libraries/stdlib/samples/test/samples/collections/maps.kt

```diff
@@ -303,6 +302,12 @@ class Maps {
             assertPrints(props.getProperty("z", "fail"), "fail")
         }
 
+        @Sample
+        fun mapMap() {
+            val peopleToAge = mapOf("alice" to 20, "bob" to 21)
+            assertPrints(peopleToAge.map { it.value }, "[20, 21]")
```

Comment:
I propose to add another example with entry argument destructuring:
`peopleToAge.map { (name, age) -> "$name is $age years old" }`

Reply:
Indeed, I placed it first as it is the obvious way to use it.